### DISCUSSION TITLE
2022-08-31 Weekly Meeting 71 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#284] [Weekly Meetings]
Weekly Meeting 71 is happening on Wednesday August 31st, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers
- Check in on MSF

### Working Group Updates:
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [OMI Website update](https://github.com/omigroup/website)
- [MSF delegates]() 
- any others?

### Discussions: 
1) TBD! Leave comments below on topics for discussion. 

### Future demo themes: (Vote now in the `#omi-vote` Discord channel!)
- `Wearables`
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Generative and AI tooling`

**Comment by mrmetaverse** [comment]
# Next weeks theme 

Let's do a `wearables` demo next week!  Thoughts? Volunteers?

**Reply by mrmetaverse** [reply]
Jin - Metaloot and more

**Reply by jackboz** [reply]
Jenn (technical artist from Gravity Layer): how we work with fashion designers to bring their designs in game engines and metaverses.

**Reply by mrmetaverse** [reply]
Thanks!

**Reply by mrmetaverse** [reply]
![WEARABLESinterop-090722](https://user-images.githubusercontent.com/63426722/188708143-53894024-cd48-4c15-8dac-dbd39bc2edde.png)

**Comment by mrmetaverse** [comment]
# Media 
- no updates on the automation side, but you are encouraged to use our automation. If you have projects or announcements, see the omi-tweets channel for examples of how we can help you get the words out 
- ex. 
![image](https://user-images.githubusercontent.com/63426722/187728199-abdd3e70-2bf5-4a64-be57-6da5a3ff78c1.png)
- next meeting:

**Comment by mrmetaverse** [comment]
# gLTF 
- Working with Khronos on pushing forward the KHR extension 
- next meeting is tomorrow

**Comment by mrmetaverse** [comment]
# Scripting 
- No updates this week :smile: 
- next meeting is tomorrow 

# Developer Hour 
- next meeting is next week on Thursday

**Comment by madjin** [comment]
## Avatar Interop Group 8/30 notes:
> This meetup group is dedicated to avatar interoperability R&D. We experiment with interoperable file formats, wearables, and animation systems between game engines and virtual worlds. While testing we also aim to provide useful feedback to other open metaverse standards working groups, such as the [Metaverse Standards Forum](https://metaverse-standards.org/), [OMI scripting group](https://github.com/omigroup/omi-scripting-group) and the [OMI glTF-extensions group](https://github.com/omigroup/gltf-extensions).

Link: https://github.com/M3-org/avatar-interop
**New time: 8pm EST / 12:00 AM UTC**

I gave a workshop on adding wearables to VRM avatars using blender (uploading notes later)
![mpv-shot0029](https://user-images.githubusercontent.com/32600939/187729046-96e80b54-d5ed-4db4-98b0-38fc8f12a1a3.jpg)

We talked about a bounty program for documentation / templates to improve open source project repos related to avatar wearables / pets / vehicles. We're starting with Webaverse first. I'm adding finishing touches later to this project today
![image](https://user-images.githubusercontent.com/32600939/187729445-c78f7318-44f5-4d69-86e7-f20de43b6fb0.png)


Later we took field trips into [Hyperfy ](https://hyperfy.io/explore) to check out their VRM avatar + WebXR + multiplayer support
Video recap: https://twitter.com/hyperfy_io/status/1564840101787471872

![brave_BqHfi89bf4](https://user-images.githubusercontent.com/32600939/187729248-9dc4227d-0a81-433d-960b-2f3e73955257.png)

**Comment by mrmetaverse** [comment]
# Website 
- add youtube link to the website: https://www.youtube.com/channel/UCB3huP95JrOTma5LfiDCd4A
- if you have other suggestions for website group please add them here: https://github.com/omigroup/website/issues
- @antpb has been working on a cool 3d tool for the site as well.

**Comment by mrmetaverse** [comment]
# MSF Delegates 
- [MSF](https://metaverse-standards.org/) oversight meeting was today. 
  - [08/31/22 Metaverse Standards Forum: Oversight (EMEA)](https://github.com/omigroup/MSF-Delegates/discussions/6)
- group has a new github repo with a readme, primarily to sync github with their discord channel. The repo is still being setup: https://github.com/omigroup/MSF-Delegates/milestone/1
- a question came up in MSF about splitting Metaverse `Fashion` away from Metaverse `Wearables`
  - What would we think of that / suggest?

**Comment by mrmetaverse** [comment]
Q: What about something like [dework](https://app.dework.xyz/bounties), do we want to use bounties? 

Update (new Milestone): https://github.com/omigroup/omigroup/milestone/5

**Reply by mrmetaverse** [reply]
challenges: setup, funding

**Reply by mrmetaverse** [reply]
gains: little extra overhead, github integration,  help with tasks/maintenance from people who are motivated by payment for services

**Reply by mrmetaverse** [reply]
![image](https://user-images.githubusercontent.com/63426722/187735703-adac3765-4f5e-4341-a507-dea14ee11cf8.png)

**Reply by mrmetaverse** [reply]
"we should do a workshop on it to dive in more"

**Reply by mrmetaverse** [reply]
"there are a variety of motivators for people aside from financial" 
![image](https://user-images.githubusercontent.com/63426722/187735489-41112311-0def-4f6f-83f1-7df755b42c3d.png)

**Reply by mrmetaverse** [reply]
New milestone created: https://github.com/omigroup/omigroup/milestone/5 

Cc: @madjin

**Comment by mrmetaverse** [comment]
Can we link OMI-announcements to events?

**Comment by mrmetaverse** [comment]
# SVVR meeting today 7pm PST
- in person event for those in the bay area

**Comment by mrmetaverse** [comment]
# OMI UX Research Pro Tem
- group is finding best time to meet on this poll: https://doodle.com/meeting/participate/id/aKZ1q8Me
- Vote is live: [FORM THE UX RESEARCH GROUP? Vote on Discord](https://discordapp.com/channels/770382203782692945/976535598313644113/1014583200145748068)
